scripts/imageProcessing/sliceImages.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+
+const universalFrameSize = 64;
+const universalSheetWidth = 832;
+
+function cropToSubSheet(sheet, width, height, offsetX, offsetY, targetDir) {
+    const index = sheet.lastIndexOf("\/");
+    var directory = "";
+    if (index > -1) {
+        directory = sheet.substring(0, index);
+    }
```

Comment:
This lastIndexOf / substring code is repeated below. Let's extract if to a function.

Reply:
yeah this script will be a one-timer, I will remove it before publishing this PR. So no need to look at it from CQ perspective. As long as it gets the job done ;)

Reply:
Fair enough.